src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_horizontal_layout.html.twig

```diff
@@ -24,10 +24,18 @@ col-sm-2
     {%- if expanded is defined and expanded -%}
         {{ block('fieldset_form_row') }}
     {%- else -%}
-    {%- set widget_attr = {} -%}
-    {%- if help is not empty -%}
-        {%- set widget_attr = {attr: {'aria-describedby': id ~"_help"}} -%}
-    {%- endif -%}
+        {%- set describedby = '' -%}
+        {%- if errors|length > 0 -%}
+            {%- set describedby = (describedby~" form-error-"~id) | trim -%}
+        {%- endif -%}
+        {%- if help is not empty -%}
+            {%- set describedby = (describedby~" "~id~"_help") | trim -%}
+        {%- endif -%}
+        {%- if describedby is not empty -%}
+            {%- set widget_attr = {attr: {'aria-describedby': describedby}} -%}
+        {%- else -%}
+            {%- set widget_attr = {} -%}
```

Comment:
Im not sure about this, but coudn't `widget_attr` be defined already? If so, this would remove that.

Reply:
I guess no: Looking at the above diff, it seems to me that previously `widget_attr` would be initialized empty as well, so this would at least not change that.